test/framework/src/main/java/org/elasticsearch/cluster/ESAllocationTestCase.java

```diff
@@ -450,14 +451,32 @@ public MockAllocationService(
             ClusterInfoService clusterInfoService,
             SnapshotsInfoService snapshotsInfoService
         ) {
-            super(
+            this(
```

Comment:
I'd rather add this constructor parameter to all the callers (there's not very many) rather than delegating to another constructor here. Could be done as a preliminary step to reduce noise.